### ISSUE TITLE
Use the pretraining model to train other datasets

### ISSUE THREAD

**Issue by ZChengLong578** [HumamAlwassel/TSP#20]
Hi, @HumamAlwassel
After downloading the pre-training model as you said, I overwrote the value of epoch to 0. The following changes were then made in the code:
![1653905168503](https://user-images.githubusercontent.com/86870600/170969695-705d2915-9203-4b90-869a-7d1675d80358.png)
![1653905194890](https://user-images.githubusercontent.com/86870600/170969760-126cf54c-1a9d-439b-a93f-76153f9526bb.png)
![1653905230207](https://user-images.githubusercontent.com/86870600/170969906-9996d3b5-9e94-48b6-b2a6-e758504bc862.png)
I would like you to take a look, is the change I made in the code correct? Or should I replace the initial tac-on-kinetics Pretrained weights with this instead of using it in the resume?

**Comment by HumamAlwassel** [comment]
This looks good to me

**Comment by ZChengLong578** [comment]
Thank you for your reply! I used the above code to train my data set and found that the accuracy rate has not changed much and has remained around 3. Here is the output of the training:
![1655111728870](https://user-images.githubusercontent.com/86870600/173321643-9daf8983-5ba0-4211-9307-ce04425fc4bc.png)
Do you know what caused it?
